src/Mimi.jl

```diff
@@ -721,6 +722,38 @@ end
 
 #End of graph section
 
-include("references.jl")
+# Begin plotting section
+
+"""
+Three defaults: single line plot, multiple line plots with legend, bar graph
+when indexing over regions, etc.
+"""
+function auto_plot(m::Model, component::Symbol, parameter::Symbol ; index::Symbol = :time, legend::Symbol = nothing, xlabel = string(index), ylabel = string(parameter), dest = nothing)
+  if isnull(m.mi)
+      run(m)
+  end
+
+  pyplot()
```

Comment:
I believe if we don't use this here, Plots.jl will auto pick some backend? That might be better, right? It would also respect any backend choice a user might have made before calling this plotting function here.